### DISCUSSION TITLE
怎么设置放大画面时候黑边字幕移入画面内

### DISCUSSION THREAD

**Discussion by jiezi506** [hooke007/mpv_PlayKit#429] [mpv 通用]
通过小键盘9放大画面的时候，黑边内的sup字幕会被抵出屏幕看不到。
以前mpc-be+xysubfilter放大画面的时候，sup到屏幕底部后会移入画面内，这样感觉更合理好一些

**Comment by hooke007** [comment]
你改了什么参数

**Comment by jiezi506** [comment]
直接下载标准版本安装使用的，什么也没有修改

**Reply by hooke007** [reply]
>什么也没有修改

我的标准版可没有小键盘9...

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/blob/0efc15e271c875be5b6df2d1f039e69b5a67f872/portable_config/mpv.conf#L308

**Reply by jiezi506** [reply]
改成yes就实现了，谢谢H大
